intersphinx: prevent resource downloads at build time\
issuetracker: broken. see bug #527188
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -28,7 +28,7 @@ sys.path.append(os.path.dirname(os.path.abspath('.')))
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.intersphinx', 'sphinxcontrib.issuetracker']
+extensions = ['sphinx.ext.autodoc']
 
 # link to sqlalchemy docs
 intersphinx_mapping = {
